src/components/Reports/ReportEdit.tsx

```diff
@@ -222,6 +231,12 @@ export default function ReportEdit({ organization }: ReportEditProps): JSX.Eleme
     }
   };
 
+  const onAddPhotoIdToRemove = (id: number) => {
```

Comment:
`onRemovePhoto` ..adding the photo id for later deletion is an internal implementation detail,, ok to skip that in the name i think